diff -up bltk/tools/analyzer/bltk_plot.sh.plot bltk/tools/analyzer/bltk_plot.sh
--- bltk/tools/analyzer/bltk_plot.sh.plot	2010-08-04 10:05:13.113949113 +0200
+++ bltk/tools/analyzer/bltk_plot.sh	2011-07-13 11:13:58.416271585 +0200
@@ -78,7 +78,7 @@ set_bltk_paths()
   return $RETCODE
 }
 
-set_bltk_root
+set_bltk_paths
 
 BLTK_GET_STAT_CMD=$BLTK_BIN/bltk_get_stat
 BLTK_CALC_CMD=$BLTK_BIN/bltk_calc
@@ -88,7 +88,7 @@ BLTK_PLOT_TMP=$BLTK_TMP/bltk_plot.$$
 ARGS_FILE=$BLTK_PLOT_TMP/bltk_plot.cmd
 ARGS_FILE_SAVED=$BLTK_PLOT_TMP/bltk_plot.cmd.saved
 
-OPTIONS="hDd:f:swx:yX:Y:t:o:12:vp:nR"
+OPTIONS="hDd:f:swx:y:X:Y:t:o:12:vp:nR"
 
 command_line()
 {
@@ -637,7 +637,7 @@ startup()
 {
 	CMD="type -p gnuplot"
 	$CMD >/dev/null 2>&1
-	[[ $? != 0 ]] && error "$CMD failed, cannot accces gnuplot program"
+	[[ $? != 0 ]] && error "$CMD failed, cannot access gnuplot program"
 
 	CMD="rm -rf $BLTK_PLOT_TMP"
 	$CMD
